cloudx/handler.go

```diff
@@ -232,6 +232,27 @@ func (h *SnakeCharmer) EnsureContext() (*AuthContext, error) {
 
 	if len(c.SessionToken) > 0 {
 		_, _ = fmt.Fprintf(h.verboseErrWriter, "You are authenticated as: %s\n", c.IdentityTraits.Email)
+		client, err := newKratosClient()
+		if err != nil {
+			return nil, err
+		}
+		_, resp, err := client.V0alpha2Api.ToSession(h.ctx).XSessionToken(c.SessionToken).Execute()
+		if resp.StatusCode == http.StatusUnauthorized || err != nil {
+			ok, err := cmdx.AskScannerForConfirmation(fmt.Sprintf("Your CLI session has expired. Do you wish to log in again with \"%s\"?", c.IdentityTraits.Email), h.stdin, h.verboseErrWriter)
+			if err != nil {
+				return nil, err
+			}
+			if ok {
+				if err := h.SignOut(); err != nil {
+					return nil, err
+				}
+				c, err := h.Authenticate()
```

Comment:
And will then here the email address be pre-filled?

Reply:
yes